src/main/java/me/zeroeightsix/kami/module/modules/movement/ElytraFlight.kt

```diff
@@ -23,290 +28,336 @@ import kotlin.math.sqrt
  * Updated by Itistheend on 28/12/19.
  * Updated by dominikaaaa on 26/05/20
  * Updated by pNoName on 28/05/20
- * Updated by Xiaro on 22/06/20
- *
- * Some of Control mode was written by an anonymous donator who didn't wish to be named.
```

Comment:
This was removed as there isn't any original Control mode code anymore, I heavily modified it in previous commits and this PR removes the code that was left